### ISSUE TITLE
Incompatibility styles with Elm-UI

### ISSUE THREAD

**Issue by aislanmaia** [halfzebra/create-elm-app#320]
Trying to use this tool `create-elm-app` to get HMR and following this [tutorial](https://korban.net/posts/elm/2019-11-17-elm-ui-introduction/) for Elm-UI, I've noticed some incompatibilities out of box in css styles. If I implement this tutorial without `create-elm-app`, the Elm-UI styles are applied just fine, however using create-elm-app and copy-pasting the code, I have issues related scrolling and height. 
Here the [tutorial demo](https://ellie-app.com/3TdcKSPxMv5a1) in Ellie.  

What should expected: the layout and styles just applied as the tutorial and demo, following the Elm-UI elements conventions.

I've deleted and removed references to main.css as well, but doesn't leave Elm-UI free to apply its styles.

**Comment by halfzebra** [comment]
Hi @aislanmaia, thanks for the feedback!

I'll look into this and will get back to you asap.

**Comment by neurot** [comment]
I have similar issues. This code

```elm
import Element exposing (..)
view model =
    layout
        []
    <|
        column
            [ Background.color (rgb 0 0 1)
            , height fill
            ]
            [ text "Hello World." ]
```

works as expected on ellie and elm reactor - I get a full-height blue box. But on a fresh create-elm-app in development-mode I get a text-height blue box. Something is off with the viewport in dev-mode.

**Comment by halfzebra** [comment]
You can fix this by adding the following CSS to `main.css`

```css
[data-elm-hot="true"] {
  height: inherit;
}
```

This probably should go into default setup.

What do you think?

**Comment by neurot** [comment]
Sorry for the late reply! Just installed 3.0.2 and it works perfeclty out of the box.

Thank you and happy hollydays!

**Comment by klazuka** [comment]
@halfzebra I just found out about this. If you hear of any more bugs related to `elm-hot`, please at-mention me. Thank you 😄

**Comment by klazuka** [comment]
Sorry for the massive delay in fixing this. I just published version 1.1.0 of [elm-hot](https://github.com/klazuka/elm-hot) which has the fix.
